settings.py

```diff
@@ -420,6 +421,10 @@ def get_locales():
     'django.contrib.staticfiles.finders.FileSystemFinder',
 )
 
+STATICFILES_DIRS = (
+    'kuma/static',
```

Comment:
Please use `path('kuma', 'static')` here to have cross platform compatible paths